content/kb/selinux-and-longhorn.md

```diff
@@ -0,0 +1,204 @@
+---
+title: "SELinux and Longhorn"
+author: Eric Weber
+draft: false
+date: 2023-11-21
+categories:
+- "instruction"
+- "selinux"
+- "rke2"
+- "k3s"
+---
+
+## Applicable versions
+
+All Longhorn versions.
+
+## Purpose
+
+The purpose of this article is to help Longhorn users understand how Longhorn typically acts in SELinux-enabled systems
+and provide them with basic commands to run to verify normal operation. It is a work in progress that may be expanded as
+Longhorn maintainers discover additional SELinux behaviors that are be of interest. It is not intended to be a guide to
```

Comment:
```suggestion
Longhorn maintainers discover additional SELinux behaviors that may be of interest. It is not intended to be a guide to
```

Reply:
This one was "may be" originally, but I decided to switch to "are". Unfortunately, I did so badly!